pkg/controller/injector/k8s_binding_injector.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 type k8sBindingInjector struct {
-	client *kubernetes.Clientset
+	client kubernetes.Interface
```

Comment:
It's a more generic interface for access as a client. We're using more features of the client now. Cool.